### PR TITLE
Interprete closure_library_path

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
c2c-template --vars vars_test.yaml --engine mako --files jsbuild/app.cfg.mako

Raise an error:

    Error: Cannot find module 'openlayers/node_modules/closure-util'

npm install does'nt seems to be called during build.

**Comment by sbrunner** [comment]
Yes it's normal the current test don't use the ngeo interface, I should do a fix in `c2c.tempate` to be able to don't have an error.

File: c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl [hunk 1]
```diff
@@ -303,3 +303,7 @@ interpreted:
     - instanceid
     - apache_entry_point
     - development
+    node:
+        vars:
+        - closure_library_path
+        cmd: ["node", "-e"]
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl]: You should add:
`ignore_error: True`
and do a rebase :-)

**Comment by ger-benjamin** [comment]
ping @arnaud-morvan 
What about this PR ?

**Comment by arnaud-morvan** [comment]
We use node modules in aeroportdelyon with GMF 1.6, so I'd like to have it in coming releases.
I've added ignore_errors and rebased on 1.6.

**Comment by sbrunner** [comment]
Looks good for me :-)

**Pull request merged**